### PR TITLE
Add-Rating

### PR DESCRIPTION
**Pull request by WangRemy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .idea
-*/__pycache__
+**/__pycache__
```
Comment (SanderBeekhuis) [inline .gitignore:2]: Nice!

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 db.sqlite3
```
Comment (SanderBeekhuis) [inline .gitignore]: Trailing newline please :)

File: song_rater/views.py [hunk 3]
```diff
@@ -15,3 +16,24 @@ def index(request):
...
+            rating.save()
+            return redirect("song_rater:song_list")
+    else:
+        form = RatingForm()
+
+    context = {"details": get_object_or_404(Song, pk=id),
+               "form": form,
+               }
+
+    return render(request, 'song_rater/song_detail.html', context)
```
Comment (SanderBeekhuis) [inline song_rater/views.py:39]: OPTIONAL: Have you looked if you can replace this with a GenricView by Django? Maybe `DetailView` would be a fit.  

I think it might lead to cleaner code, but I think you will touch this code again in follow-up Pull Requests. It might be best to keep this as-is and maybe try to refactor it at the end. :)

**Review by SanderBeekhuis — CHANGES_REQUESTED** [review]
The code you've written looks good. Nice merge request.

It might be nice to add the Rating to the admin interface as well (but this can be part of a new MR as well) 

I'm missing tests (or an argumentation why there are none). Please write some.

**Comment by SanderBeekhuis** [comment]
> It might be nice to add the Rating to the admin interface as well (but this can be part of a new MR as well)

It seems you did in #6  , fine by me.

File: song_rater/views.py (hunk 3 above)
Comment (SanderBeekhuis) [reply song_rater/views.py:39]: As discussed in a call @WangRemy  won't look into this.

**Review by SanderBeekhuis — APPROVED** [review]

**Comment by SanderBeekhuis** [comment]
Nice test case! :)
